src/service/stables.ts

```diff
@@ -56,16 +51,8 @@ export default async function stables(): Promise<TokenModel[]> {
       value = prices.value[tokenData.iso4217] * tokenData.units.value
 
       if (tokenData.symbol === StableToken.cUSD) {
-        value -= mobiusCUSDAmount.value * prices.value[tokenData.iso4217]
-        tokenData.units.value -= mobiusCUSDAmount.value
-
-        // TODO: add curveCUSDAmount back in
-
-        // value -= curveCUSDAmount.value * prices.value[tokenData.iso4217]
-        // tokenData.units.value -= curveCUSDAmount.value
-
-        value -= 10_000_000
-        tokenData.units.value -= 10_000_000
+        value -= curveCUSDAmount.value * prices.value[tokenData.iso4217]
+        tokenData.units.value -= curveCUSDAmount.value
       }
     } catch (e) {
```

Comment:
This is a good observation, but imo there's not much else that can be done here. As we're reading these values from the blockchain & API's, if there is a time when they cannot be retrieved, we'll have bigger problems to worry about. We could log a message but idt this will provide any value.

Reply:
let's log a message, it could be more helpful. What do you think?

Reply:
I will leave it as it is. Adding a message here provides no value. In the rare case the data sources are unavailable we have errors that will bubble up before this point that are more meaningful.

Reply:
alright 🤝

Reply:
We could forward it to sentry tbh, seeing that we have that setup.

Reply:
Let's talk about this. From my perspective a message here adds nothing.